### PR TITLE
Fix AsJob parameter description and http to https

### PR DESCRIPTION
**Pull request by velkovb**
Fixed parameter description for AsJob. Fixed http links in CimSession parameter and common parameters section.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

The PR introduces surface level fixes to a group of articles.

Note that the parameter description for -AsJob parameter has been transferred from another version of the corresponding article.

Peer review is requested.

File: docset/windows/deduplication/New-DedupSchedule.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ This job processes data corruptions it finds during data integrity validation, p
 This job undoes data deduplication on all of the optimized files on the volume. 
 At the end of a successful unoptimization job, the server deletes all of the data deduplication metadata from the volume.
 
-For more information, see [Install and Configure Data Deduplication](http://technet.microsoft.com/en-us/library/hh831434.aspx) on TechNet.
+For more information, see [Install and Configure Data Deduplication](https://technet.microsoft.com/en-us/library/hh831434.aspx) on TechNet.
```
Comment (JohanFreelancer9) [inline docset/windows/deduplication/New-DedupSchedule.md]:
```suggestion
For more information, see [Install and Configure Data Deduplication](https://technet.microsoft.com/library/hh831434.aspx) on TechNet.
```

File: docset/winserver2012-ps/deduplication/New-DedupSchedule.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ You can create a schedule to run the following types data deduplication jobs:
 - Scrubbing: This job processes data corruptions it finds during data integrity validation, performs possible corruption repair, and generates a scrubbing report. 
 - Unoptimization: This job undoes data deduplication on all of the optimized files on the volume. At the end of a successful unoptimization job, the server deletes all of the data deduplication metadata from the volume.
 
-For more information, see Install and Configure Data Deduplicationhttp://technet.microsoft.com/en-us/library/hh831434.aspx (http://technet.microsoft.com/en-us/library/hh831434.aspx) on TechNet.
+For more information, see [Install and Configure Data Deduplication](https://technet.microsoft.com/en-us/library/hh831434.aspx) on TechNet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/deduplication/New-DedupSchedule.md]:
```suggestion
For more information, see [Install and Configure Data Deduplication](https://technet.microsoft.com/library/hh831434.aspx) on TechNet.
```

File: docset/winserver2012r2-ps/deduplication/New-DedupSchedule.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ You can create a schedule to run the following types data deduplication jobs:
 - Scrubbing: This job processes data corruptions it finds during data integrity validation, performs possible corruption repair, and generates a scrubbing report. 
 - Unoptimization: This job undoes data deduplication on all of the optimized files on the volume. At the end of a successful unoptimization job, the server deletes all of the data deduplication metadata from the volume.
 
-For more information, see Install and Configure Data Deduplicationhttp://technet.microsoft.com/en-us/library/hh831434.aspx (http://technet.microsoft.com/en-us/library/hh831434.aspx) on TechNet.
+For more information, see [Install and Configure Data Deduplication](https://technet.microsoft.com/en-us/library/hh831434.aspx) on TechNet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/deduplication/New-DedupSchedule.md]:
```suggestion
For more information, see [Install and Configure Data Deduplication](https://technet.microsoft.com/library/hh831434.aspx) on TechNet.
```

**Pull request merged**